rviz_common/src/rviz_default_plugins/point_cloud2_display.cpp

```diff
@@ -27,42 +27,35 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "point_cloud2_display.hpp"
+
 #include <memory>
+#include <utility>
 
 #include <OgreSceneNode.h>
 #include <OgreSceneManager.h>
 
-#include "./point_cloud_common.hpp"
-#include "./point_cloud_transformers.hpp"
+#include "point_cloud_common.hpp"
+#include "point_cloud_helpers.hpp"
 #include "rviz_common/display_context.hpp"
 #include "rviz_common/frame_manager.hpp"
 #include "rviz_rendering/point_cloud.hpp"
 #include "rviz_common/properties/int_property.hpp"
 #include "rviz_common/validate_floats.hpp"
 
-#include "point_cloud2_display.hpp"
-
 namespace rviz_default_plugins
 {
 
 PointCloud2Display::PointCloud2Display()
-: point_cloud_common_(new PointCloudCommon(this))
+: point_cloud_common_(std::make_unique<PointCloudCommon>(this))
```

Comment:
This is redundant, it does the same thing as the original code and actually requires an additional move for the unique ptr (the temporary returned by make unique needs to be moved into the storage).